config/config.exs

```diff
@@ -32,3 +32,9 @@ config :phoenix, :format_encoders,
 config :mime, :types, %{
   "application/vnd.api+json" => ["json-api"]
 }
+
+config :guardian, Guardian,
+  issuer: "MyApp",
```

Comment:
Oh, whoops! Gotta change that to `Streamr`. I'll do that then fix merge conflicts.